### ISSUE TITLE
Add competition pages

### ISSUE THREAD

**Issue by coder13** [cubingusa/org-old#5]
Grab all the txt files for each competition

Create a template for the competition with a home page, events, schedule and list of competitors

**Comment by jbmertens** [comment]
Is the goal here to preserve content from old competition websites? At one point I was thinking about just creating a mirror using wget, something like [this](https://www.guyrutenberg.com/2014/05/02/make-offline-mirror-of-a-site-using-wget/), and tailoring the content some (eg, contact pages).

A big factor in that was due to old websites being fairly inhomogeneous. For example, nats websites contain custom pages, old ca/l/tech websites don't have .txt files, etc.

**Comment by coder13** [comment]
Jeremy had to leave when I was in the middle of discussing this so I could be in the wrong  here.

mirror? Will we still have the server running to serve up the pages? Jeremy said he'd like to go full static.

I'm not entirely sure about pages like nats but I'm sure there's a way to do it.

My thinking in this was that since Jekyll supports collections and being able to store pages like this as just stripped down information and that's how all the competition websites are, it would be the simplest way to move over and store the least amount of data.

**Comment by jbmertens** [comment]
Re: mirror, not a proxy or anything, just a wget mirror, which stores page html + assets. It isn't that efficient in terms of storage, but shouldn't be too terrible... couple hundred MB?

**Comment by jfly** [comment]
Yeah, I had to run before answering Caleb about this. I think just a simple wget static-fication as Jim suggests is the best way to go. Maybe try to consolidate repeated image assets? I think the only limiting factor here is git repository size.

**Comment by jbmertens** [comment]
If we want to do the wget thing, I'll volunteer to do the gruntwork, consolidating assets, making sure we have all websites, etc.
